.gitignore

```diff
@@ -7,4 +7,5 @@ resources/implementationguide/ig-*.*
 org.hl7.*.jar
 publisher.jar
 publisher.jar.bak
-ch.fhir.ig.ch-orf.xml
\ No newline at end of file
+ch.fhir.ig.ch-orf.xml

```

Comment:
why ignoring this file?

Reply:
I don't remember why this file is in gitignore. Anyway, it does no harm, because it is not used. I will remove it.